config/logging.yaml

```diff
@@ -49,15 +59,36 @@
 #     [%X{session}] session ID from cookies, else the request-ID
 #     [%X{session_safe}] random ID assigned per session, not available early on during
 #                        request logging, but prevents disclosure of the session ID
-#     [%X{string}] variable set using ::Logging.mdc['string'] =
-
+#     [%X{string}] variable set via MDC (::Logging.mdc['string'])
+#     [%x{string}] variable set via NDC (::Logging.ndc['string'])
+#
+# :sys_pattern:
+#   Logger line pattern for system log/journal. See :pattern for more info.
+#
+# :facility:
+#   System log/journal facility to use. One of LOG_AUTH, LOG_AUTHPRIV, LOG_CONSOLE,
+#   LOG_CRON, LOG_DAEMON, LOG_FTP, LOG_KERN, LOG_LPR, LOG_MAIL, LOG_NEWS, LOG_NTP,
+#   LOG_SECURITY, LOG_SYSLOG, LOG_USER, LOG_UUCP, LOG_LOCAL0 to LOG_LOCAL7. See
+#   syslog(3) man page for more info.
+#
 :default:
   :colorize: false
   :console_inline: false
   :log_trace: false
   :level: info
   :type: file
-  :pattern: "%d %.8X{request} [%c] [%.1l] %m\n"
+  #:layout: multiline_pattern
+  :layout: json
```

Comment:
I'd rather default to multiline & journald, isn't it?

Reply:
Default value must be preserved as `file` for easy development setup. What will installer set is still a question tho. @ekohl and @ares discussed that, me as well. I'd love to see journald to be the production default, but there are concerns with puppet.